_episodes/13-ides.md

```diff
@@ -66,116 +66,104 @@ and some can also interact with a version control system.
-   the current Python interpreter that is being used.
-   These may be blank or set to `<No interpreter>`,
-   or possibly the default version of Python installed on your system,
-   e.g. `Python 2.7 /usr/bin/python2.7`,
-   which we do not want to use in this instance.
-3. Select the cog-like button in the top right, then `Add...` 
-   (or `Add Local...` depending on your PyCharm version).
-   An `Add Python Interpreter` window will appear.
-4. Select `Virtualenv Environment` from the list on the left
-   and ensure that `Existing environment` checkbox is selected within the popup window.
-   In the `Interpreter` field point to the Python 3 executable inside 
-   your virtual environment's `bin` directory
-   (make sure you navigate to it and select it from the file browser rather than
-   just accept the default offered by PyCharm).
-   Note that there is also an option to create a new virtual environment, 
-   but we are not using that option as we want to reuse the one we created
-   from the command line in the previous episode.
-   ![Configuring Python Interpreter in PyCharm](../fig/pycharm-configuring-interpreter.png){: .image-with-shadow width="800px"}
-5. Select `Make available to all projects` checkbox
-   so we can also use this environment for other projects if we wish.
-6. Select `OK` in the `Add Python Interpreter` window.
-   Back in the `Preferences` window, you should select "Python 3.9 (python-intermediate-rivercatchment)"
-   or similar (that you've just added) from the `Project Interpreter` drop-down list.
-
-Note that a number of external libraries have magically appeared under the
-"Python 3.9 (python-intermediate-rivercatchment)" interpreter,
-including `numpy` and `matplotlib`.
-PyCharm has recognised the virtual environment we created from the command line using `venv`
-and has added these libraries effectively replicating our virtual environment in PyCharm
-(referred to as "Python 3.9 (python-intermediate-rivercatchment)"). 
-
-![Packages Currently Installed in a Virtual Environment in PyCharm](../fig/pycharm-installed-packages.png){: .image-with-shadow width="800px"}
+
+![View of an opened project in VS Code](../fig/vs-code-open-project.png){: .image-with-shadow width="1000px" }
+
+You may have noticed that the virtual environment (`venv`) that we created previously
+(held in the `./venv` folder in our project directory)
+is now showing on the bottom status bar, on the right hand side.
+If this is the case, it shows that VS Code is using that environment as the interpreter.
+In many cases, VS Code will have been able to locate this environment automatically,
+and add it as the most likely option (as it is located within the current project directory).
+But it is better to explicitly select an interpeter for your project,
+to do this follow these steps:
+1. Press `Command-Shift-P` (on Mac) or `Control-Shift-P` (on Windows) to bring up the VS Code command palette
+2. Search for `select interpreter` and select it
```

Comment:
Should this be: `Python: select interpreter`

Reply:
Yes, I'll update to that. (Although just `select interpreter` will work, but we should avoid confusion)

Reply:
I mostly mentioned it because when is searched "select interpreter" it also comes up with the jupyter intepreter, but yes users should be able to work out what is meant.

Reply:
@Scottan have done the above correction